graphql-java-support/src/main/java/com/apollographql/federation/graphqljava/directives/LinkDirectiveProcessor.java

```diff
@@ -36,7 +38,10 @@ public final class LinkDirectiveProcessor {
           "@context", 28,
           "@fromContext", 28,
           "@cost", 29,
-          "@listSize", 29);
+          "@listSize", 29,
+          "@cacheTag", 212);
```

Comment:
since we updated the logic below -> all those numbers should be updated to reflect it (i.e. 21 -> 201, etc)